osx/src/main.cpp

```diff
@@ -278,6 +278,7 @@ void init_main_window(bool recreate) {
 
     // Setup tangram
     Tangram::initialize(sceneFile.c_str());
```

Comment:
Discussed IRL with @tallytalwar : we can have `Tangram::initialize` taking no scene path now, since it's not loading the scene anymore. In that case it would be a dummy initialization for the main objects accessing that does not access  any resources.